USAGE.rst

```diff
@@ -7,13 +7,34 @@ See README.rst for installation.
 Importing Data
 ==============
 
-TODO Post-Beta: clean this up when the process is worked out
+The import process assumes that the system has been configured with a mounted import directory, that is, the local filesystem folder that was mounted as the import path in the docker-compose configuration.
+
+Imported Files and Folders
+--------------------------
+
+Files are copied from the local import directory to the ``Imported`` folder in the ``SEER`` collection in the DSA. Files can have any folder structure and that folder structure will be reflected in the ``Imported`` folder in the DSA. Excel files (identified by ending in .xls or .xlsx) and image files (anything else except for ignored files) will be imported. To facilitate bulk uploads, we ignore files ending in .txt, .xml, .zip from the import process--this list can be easily changed.
```

Comment:
The folder structure doesn't get reflected in the import process.  We always rename and refolder the files as part of the import process with a folder name of `TokenID` and a filename of `ImageID.<extension>`.

Let's change this line to:
`Files are copied from the local import directory to the ``Imported`` folder in the ``SEER`` collection in the DSA. Files can have any folder structure; the folder structure is not significant in the import process. Excel files (identified by ending in .xls or .xlsx) and image files (anything else except for ignored files) will be imported. To facilitate bulk uploads, we ignore files ending in .txt, .xml, .zip from the import process -- this list can be easily changed.`